<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Istok_relevants</class>
 <widget class="QMainWindow" name="Istok_relevants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>250</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Istok_relevants</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PAGE</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="PageNo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Discript1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>175</width>
      <height>175</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Discript2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>175</width>
      <height>175</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Name1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>175</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Name2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>175</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
